vulcanai2/models/__init__.py

```diff
@@ -1,5 +1,6 @@
 from .basenetwork import BaseNetwork
 from .cnn import ConvNet
 from .dnn import DenseNet
+from .utils import *
```

Comment:
Why do we have this? I'm worried about "polluting the namespace", especially if our utility functions are generically named.

Reply:
that is a good point. @priyatharsan would you be able to specify what from utils you require so we can be more specific with the imports?

Reply:
also on that note, I think what is found in utils should be tied to BaseNetwork since intuitively you should be able to do model.get_size and such instead of passing a model to get_size(model) and get_output_shapes(model)? I'm curious to know your thoughts on this matter

Reply:
Oh I didn't notice that. Good point, I agree completely, those should be with BaseNetwork.

Reply:
removed the contents of utils to BaseNetwork as it makes more sense there

Reply:
feel free to push your commit to this branch so we can look at the updated version